segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/LogReader.java

```diff
@@ -75,12 +77,7 @@
     public void close() {
         if (!this.closed.getAndSet(true)) {
             if (this.currentLedger != null) {
-                try {
-                    Ledgers.close(this.currentLedger.handle);
-                } catch (DurableDataLogException bkEx) {
-                    log.error("Unable to close LedgerHandle for Ledger {}.", this.currentLedger.handle.getId(), bkEx);
-                }
-
+                this.currentLedger.close();
```

Comment:
Why did you remove the try-catch here but not below (at 237)?

Reply:
@andreipaduroiu 
because at line 237 we have
`Ledgers.close(xxx)`
that may throw an exception, and it is actually not possible that it happens

here we are using currentLedger.close(); (LogReader#close)) that calls exactly the code at line 237, and there is not need to add a catch clause.